File: mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
 int h;
 do
    {
    printf("Height:");
    h = get_int();
    } while (h<0 || h >23);
    {
        int n=h-1;
        for (int k=0; k<h; k++)
        {
            for(int i=n-k; i>=0; i--)
            {
                printf(" ");
            }
            for( int j= 0; j<k+2; j++)
            {
                printf("#");
            }
        
        printf("\n");
        }
    }
    
}

File: mario.py
import cs50

def main():
    while True:
        print("Height: ", end = "")
        h = cs50.get_int()
        if h >= 0 and h <= 23:
            break
        
    for i in range(h):
        for j in range(h - i - 1):
            print(" ", end = "")
        for k in range(i+2):
            print("#", end = "")
        print("")
            
if __name__ == "__main__":
    main()